skrobot/interfaces/_pybullet.py

```diff
@@ -291,6 +295,9 @@ def wait_interpolation(self, thresh=0.05):
                     wait = True
             if wait is False:
                 break
+            if time.time() - start > timeout:
+                return False
+            time.sleep(1)
```

Comment:
Why did you add `time.sleep(1)`?

Reply:
sorry, I dont't need it 
I thought I need to add `time.sleep(1)` to make 1-loop  loop in 1sec

Reply:
OK. please delete it.